### ISSUE TITLE
A question about loss and results

### ISSUE THREAD

**Issue by HYC01** [Cocofeat/EyeMoSt#4]
Thanks for your impressive work EyeMoSt and EyeMoSt+ in MICCAI2023 and MIA 2024.
When I reproduced the work based on this code, I found that the calculated loss would be negative.
![image](https://github.com/user-attachments/assets/00e32ea5-1017-4721-95dc-a2a250b4f2e7)
And according to the method of model selection, the final weights are saved, and the results of the inference are different from the experimental results.

**Comment by Cocofeat** [comment]
Yes, loss will become negative because of the NIG distribution, which will cause a negative situation. You can refer to the following two papers:
1. Deep evidential regression
2. Trustworthy multimodal regression with mixture of normal-inverse gamma distributions

By the way, what did you test the results on?

**Comment by HYC01** [comment]
Thanks for your reply.
I do test on GAMMA dataset by five-fold cross-validation. And here is my results using model selection.
![image](https://github.com/user-attachments/assets/c1f57378-e07a-4770-8c8c-242b0a203cbe)

**Comment by HYC01** [comment]
By the way, the validation set for the best model chosen seems to be the same as the test set.
